### PR TITLE
Twenty Twenty-Three: Sync latest changes from TT3 repo (RC1)

### PR DESCRIPTION
**Pull request by mikachan**
This PR syncs the latest changes from the Twenty Twenty-Three repo (https://github.com/WordPress/twentytwentythree), ahead of RC1.

Trac ticket: https://core.trac.wordpress.org/ticket/56383

cc. @desrosj

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-content/themes/twentytwentythree/styles/marigold.json [hunk 1]
```diff
@@ -1,305 +1,317 @@
 {
   "$schema": "https://schemas.wp.org/trunk/theme.json",
-  "title": "Marigold",
+  "version": 2,
+	"title": "Marigold",
```
Comment (desrosj) [inline src/wp-content/themes/twentytwentythree/styles/marigold.json:4]: Looks like the indentation is off here. I can fix this pre-commit, and we can sync the change back to WordPress/twentytwentythree after.

**Review by desrosj — APPROVED** [review]
Thanks @mikachan!

**Comment by desrosj** [comment]
Merged into Core in https://core.trac.wordpress.org/changeset/54467.

File: src/wp-content/themes/twentytwentythree/styles/marigold.json (hunk 1 above)
Comment (mikachan) [reply src/wp-content/themes/twentytwentythree/styles/marigold.json:4]: Thank you for picking this up!

**Pull request closed without merging**